projectile.el

```diff
@@ -1257,19 +1261,22 @@ Raise an error if their is no dirty project."
             (setq status (cons (list project tmp-status) status))))))
     (when (= (length status) 0)
       (message "No dirty projects have been found"))
+    (setq projectile-cached-dirty-projects-status status)
     status))
 
-(defun projectile-browse-dirty-projects ()
-  "Browse dirty version controlled projects."
-  (interactive)
-  (let ((status nil)
+(defun projectile-browse-dirty-projects (cached)
```

Comment:
I'd make this an optional param, so you wouldn't have to pass it explicitly in non-interactive calls. In general I wonder do you need this as a param, or a configuration option.

Reply:
Good point, I passed `cached` as an optional parameter.

I am not sure what you mean by passing this as a configuration option.
You mean making it a customizable variable ?
If it is the case, I am not sure this would be a good idea, cached dirty project list is only useful when you remember what modifications you have made since the previous dirty project list update (and update is only made when `cached` is nil).